src/controllers/connectionManager.ts

```diff
@@ -270,7 +270,7 @@ export default class ConnectionManager {
         listParams.ownerUri = connectionUri;
         listParams.includePassword = includePassword;
         const result = await this.client.sendRequest(ConnectionContracts.GetConnectionStringRequest.type, listParams);
-        return result.connectionString;
+        return result;
```

Comment:
You can just return the sendRequest call, that way you aren't having to unnecessarily re-wrap the result in a promise.